ddtrace/appsec/_iast/taint_sinks/_base.py

```diff
@@ -119,12 +125,8 @@ def report(cls, evidence_value: Text = "", dialect: Optional[Text] = None) -> No
             file_name = None
             line_number = None
 
-            skip_location = getattr(cls, "skip_location", False)
-            if not skip_location:
-                frame_info = get_info_frame(CWD)
-                if not frame_info or frame_info[0] == "" or frame_info[0] == -1:
-                    return None
-
+            frame_info = get_info_frame(CWD)
```

Comment:
why not keep the skip location check before calling `get_info_frame`? That way we don't call if not needed? Being `get_info_frame` expensive in CPU I would try to avoid calling it.